app/models/noise.rb

```diff
@@ -24,13 +24,76 @@ class Noise < ActiveRecord::Base
     using: {tsearch: {dictionary: "english"}},
     associated_against: { user: :name }
 
-  delegate :url_helpers, to: 'Rails.application.routes'
-
   belongs_to :user
   has_one :original, as: :importable, dependent: :destroy
   has_many :origins, ->{ uniq true },
     dependent: :destroy
 
+  scope :where_nearby, ->(params) do
+    location = params[:location]
+    latitude = params[:latitude]
+    longitude = params[:longitude]
+
+    if latitude && longitude
+      search_location = [latitude, longitude]
+    elsif location
+      search_location = location
+    end
+
+    if search_location
+      Rails.logger.debug "Location detected " + search_location.to_s
+
+      noise_ids = Origin.explore(params)
+        .where_since(params[:created_at])
+        .map &:noise_id
+
+      where_ids(noise_ids).order_by_ids(noise_ids)
+    else
+      scoped
+    end
+  end
+
+  scope :where_needs_triage, ->(params) do
```

Comment:
Unused block argument - `params`. If it's necessary, use `_` or `_params` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.